docs/ilm.asciidoc

```diff
@@ -61,29 +61,43 @@ apm-server:
+e.g. `apm-7.9.0-span-foo`.
 The related ILM policy can specify a rollover action, e.g. when an index reaches a size of 50GB
-it should roll over to create a new index. The `rollover_alias` abstracts away the concrete write index, while
+it should roll over to create a new index. The rollover alias abstracts away the concrete write index, while
 automatically being updated and keeping track of the current write index.
-This allows to use the `rollover_alias` instead of specific indices in queries.
+This allows to use the rollover alias instead of specific indices in queries.
 
-For example when defining a rollover alias with name `apm-7.8.0-transaction`, an alias pointing to a write index
-`apm-7.8.0-transaction-000001` is created. When the index reaches the defined size or age it will roll over to
+For example the default rollover alias for event type `transaction` would be `apm-7.8.0-transaction` and points
+to a write index `apm-7.8.0-transaction-000001`. When the index reaches the defined size or age it will roll over to
 a new index `apm-7.8.0-transaction-000002`. The rollover alias `apm-7.8.0-transaction` keeps
 track of which index is the write index when ingesting data.
 
-NOTE: Ensure that the configured `rollover_alias` for all event types start with the same prefix,
-and that this prefix is also configured for `setup.template.name` and `setup.template.pattern`.
-If the template configuration is not accordingly changed, the Elasticsearch index template with
-the predefined mappings will not match against created indices, leading to indexing issues.
+NOTE: Manually ensure that templates containing customized index information do not conflict with each other
+or the default templates.
 
-The example below shows how to change the `rollover_alias` to a custom value,
-and set the template configuration to a matching name and pattern.
+By default the APM Server creates a template without a custom index suffix per event type. When defining custom
+index suffixes, always ensure that templates, that might have been set up previously, are removed or do not conflict.
+A conflicting behavior could occur when an index matches multiple templates with the same `order`.
+For example, the APM Server was started without any customization, leading to a default index setup. Afterwards the
+Server configuration is customized for the event type `span` to set up a custom index suffix `production`.
+When restarting the Server it will set up the new index template considering the custom part.
+A newly created index `apm-server-7.9.0-span-production` would now match the default template with the index pattern
+`apm-server-7.9.0-span*` but also the new template with the index pattern `apm-server-7.9.0-span-production*`.
+In such a case the old template needs to be manually deleted,
+or the `index_pattern` or the `order` need to be changed to avoid conflicts.
```

Comment:
```suggestion
For example, the APM Server was started without any customization, leading to a default index setup. Afterward, the
Server configuration is customized to add the index suffix `production` for the event type `span`.
When the Server restarts, it will set up a new index template based on the new custom index suffix.
A newly created index named `apm-server-7.9.0-span-production` would now match the default template with the index pattern of
`apm-server-7.9.0-span*`, but also the new template with the index pattern `apm-server-7.9.0-span-production*`.
In this case, the old template needs to be manually deleted,
or the `index_pattern` or `order` need to be changed to avoid conflicts.
```